### ISSUE TITLE
Remove hard-coded "metadata" in FindBugsLauncher#buildFakePluginJar

### ISSUE THREAD

**Issue by Kidlike** [find-sec-bugs/find-sec-bugs#479]
## Description
Forgive any ignorance in the _findbugs-verse_.
When writing a findbugs plugin, I based my tests on `BaseDetectorTest` and `EasyBugReporter`.

During the execution of the test this method is hit in the stacktrace: `com.h3xstream.findbugs.test.service.FindBugsLauncher#buildFakePluginJar`

It seems that this method is a bit opinionated on maven structure. Specifically [this line](https://github.com/find-sec-bugs/find-sec-bugs/blob/master/findbugs-test-util/src/test/java/com/h3xstream/findbugs/test/service/FindBugsLauncher.java#L153), that depends on the `"metadata"` folder:

```
final URL metadata = cl.getResource("metadata");
if (metadata != null) {
    final File dir = new File(metadata.toURI());
    
    //Add files to the jar stream
    addFilesToStream(cl, jar, dir, "");
}
```

If I'm not mistaken this folder with its contents is required for running these tests.
Is it possible to make this `"metadata"` not hard-coded?

For example add a second [constructor in BaseDetectorTest](https://github.com/find-sec-bugs/find-sec-bugs/blob/master/findbugs-test-util/src/test/java/com/h3xstream/findbugs/test/BaseDetectorTest.java#L50), which could pass-on the argument to `FindBugsLauncher`.

## Code
I'm using a test structure that I saw in this project:
```
public class MyTest extends BaseDetectorTest {
    private EasyBugReporter reporter;

    @Before
    public void setup() {
        reporter = spy(new EasyBugReporter());
    }

    @Test
    public void aTest() throws Exception {
        // given
        final String[] files = {
            getClassFilePath("samples/SomeTestStub")
        };

        // when
        analyze(files, reporter);

        // then
        // assertions...
    }
}
```

**Comment by h3xstream** [comment]
Yep. This is definitely possible.

**Comment by h3xstream** [comment]
Thanks for the contrib.
Merged #480
